python/cxx/export.hpp

```diff
@@ -23,18 +24,21 @@ void export_FieldProperties(py::module& module);
 void export_EclipseConfig(py::module& module);
 void export_EclipseGrid(py::module& module);
 void export_EclipseState(py::module& module);
+void export_EclipseState_embedded(py::module& module);
 void export_Group(py::module& module);
 void export_ParseContext(py::module& module);
 void export_Parser(py::module& module);
 void export_SimulationConfig(py::module& module);
 void export_Schedule(py::module& module);
+void export_Schedule_embedded(py::module& module);
 void export_ScheduleState(py::module& module);
 void export_TableManager(py::module& module);
 void export_Well(py::module& module);
 void export_Log(py::module& module);
 void export_IO(py::module& module);
 void export_EModel(py::module& module);
 void export_SummaryState(py::module& module);
+void export_SummaryState_embedded(py::module& module);
```

Comment:
Is there a formal requirement that all functions which export something have to be named `export_*`?  If not, could we conceivably have
```C++
namespace python::common::export {
void SummaryState_embedded(py::module& module);
// &c
}
```
instead?

Reply:
export is a keyword in C++, if we use another one, like python::common::export_opm then that's possible